cmd/ore/azure/azure.go

```diff
@@ -42,6 +43,7 @@ func init() {
 
 	sv := Azure.PersistentFlags().StringVar
 	sv(&azureProfile, "azure-profile", "", "Azure Profile json file")
+	sv(&azureAuth, "azure-auth", "", "Azure auth location")
```

Comment:
can we default this to something like `.azure/credentials.json`. I don't really care what, but we should have a canonical location so kola runs with azure don't need special args/envs